src/sider/oppfolgingsplan/oppfolgingsplaner/AktiveOppfolgingsplaner.tsx

```diff
@@ -0,0 +1,153 @@
+  ingenAktiveOppfolgingsplaner: "Det er ingen aktive oppfølgingsplaner",
+  foresporselSendt: "Forespørsel om oppfølgingsplan ble sendt",
+  aktivForesporsel:
+    "Obs! Det ble bedt om oppfølgingsplan fra denne arbeidsgiveren",
+};
+
+function activeNarmesteLederForCurrentOppfolgingstilfelle(
+  ledere: NarmesteLederRelasjonDTO[],
+  oppfolgingsTilfelle: OppfolgingstilfelleDTO
+): NarmesteLederRelasjonDTO[] {
+  return ledere.filter(
+    (leder) =>
+      leder.status === "INNMELDT_AKTIV" &&
+      oppfolgingsTilfelle.virksomhetsnummerList.includes(
+        leder.virksomhetsnummer
+      )
+  );
+}
+
+interface Props {
+  aktivePlaner: OppfolgingsplanDTO[];
+  oppfolgingsplanerLPSMedPersonoppgave: OppfolgingsplanLPSMedPersonoppgave[];
+}
+
+export default function AktiveOppfolgingsplaner({
+  aktivePlaner,
+  oppfolgingsplanerLPSMedPersonoppgave,
+}: Props) {
+  const { toggles } = useFeatureToggles();
+  const { currentLedere } = useLedereQuery();
+  const getOppfolgingsplanForesporsel = useGetOppfolgingsplanForesporselQuery();
+  const postOppfolgingsplanForesporsel = usePostOppfolgingsplanForesporsel();
+  const { latestOppfolgingstilfelle, hasActiveOppfolgingstilfelle } =
+    useOppfolgingstilfellePersonQuery();
+  const currentOppfolgingstilfelle = hasActiveOppfolgingstilfelle
+    ? latestOppfolgingstilfelle
+    : undefined;
+
+  const oppfolgingsplanerLPSUnprocessed = oppfolgingsplanerLPSMedPersonoppgave
+    .filter((oppfolgingsplanLPS) => {
+      if (oppfolgingsplanLPS.personoppgave) {
+        if (oppfolgingsplanLPS.personoppgave.behandletTidspunkt) {
+          return (
+            Date.now() <
+            dayjs(oppfolgingsplanLPS.personoppgave.behandletTidspunkt)
+              .add(1, "days")
+              .toDate()
+              .getTime()
+          );
+        }
+        return !oppfolgingsplanLPS.personoppgave.behandletTidspunkt;
+      }
+      return erIdag(oppfolgingsplanLPS.opprettet);
+    })
+    .sort((a, b) => {
+      return new Date(b.opprettet).getTime() - new Date(a.opprettet).getTime();
+    });
+
+  const hasActivePlan =
+    aktivePlaner.length !== 0 || oppfolgingsplanerLPSUnprocessed.length !== 0;
+
+  const activeNarmesteLedere = !!currentOppfolgingstilfelle
+    ? activeNarmesteLederForCurrentOppfolgingstilfelle(
+        currentLedere,
+        currentOppfolgingstilfelle
+      )
+    : [];
+  const activeNarmesteLederIfSingle =
+    activeNarmesteLedere.length === 1 ? activeNarmesteLedere[0] : undefined;
+  const isBeOmOppfolgingsplanVisible =
+    toggles.isBeOmOppfolgingsplanEnabled &&
+    !hasActivePlan &&
+    !!currentOppfolgingstilfelle &&
+    !!activeNarmesteLederIfSingle &&
+    !postOppfolgingsplanForesporsel.isSuccess;
+
+  const lastForesporselCreatedAt =
+    getOppfolgingsplanForesporsel.data?.[0]?.createdAt;
+
+  const isAktivForesporsel =
+    !!lastForesporselCreatedAt &&
+    !!currentOppfolgingstilfelle &&
+    !postOppfolgingsplanForesporsel.isSuccess
```

Comment:
Skjønner ikke helt hvorfor `isAktivForesporsel` skal være `false` når man har gjort et suksess-kall for å be om oppfølgingsplan :thinking:

Reply:
Hm, kanskje den bør hete `isExistingForesporsel`? 🤔
Det er fordi den warning alerten ikke skal vises når man akkurat har sendt avgårde en ny forespørsel. For da skal bare det grønne banneret vises.
Synes også det er litt rar flyt med hva som vises når her.

Reply:
Hmja, skjønner. Kan jo få på noen tester på logikken for hva som vises når først, så evt prøve å refaktorere/forenkle logikken etterpå.